src/ViewModel/Converter/PressPackageContentHeaderConverter.php

```diff
@@ -27,26 +24,32 @@ public function __construct(UrlGeneratorInterface $urlGenerator)
      */
     public function convert($object, string $viewModel = null, array $context = []) : ViewModel
     {
-        $subjects = $object->getSubjects()->map(function (Subject $subject) {
-            return new ViewModel\Link($subject->getName(), $this->urlGenerator->generate('subject', [$subject]));
-        })->toArray();
-
-        return new ViewModel\ContentHeader($object->getTitle(), null, $object->getImpactStatement(), true,
-            null, $subjects, null, null, null,
-            new ViewModel\SocialMediaSharers(
-                strip_tags($object->getTitle()),
-                $this->urlGenerator->generate('press-pack', [$object], UrlGeneratorInterface::ABSOLUTE_URL)
-            ),
+        if ($date = $this->simpleDate($object, ['date' => 'published'] + $context)) {
```

Comment:
the `else` is not needed here.  e.g.
```
$meta = null;
if ($date = $this->simpleDate($object, ['date' => 'published'] + $context)) {
    $meta = ViewModel\MetaNew::withDate($date);
}
```

Reply:
We need to decalare it if the condition is false, otherwise it will trow an error.

Reply:
Done.